brave/utility/tor/tor_launcher_impl.cc

```diff
@@ -147,6 +147,8 @@ void TorLauncherImpl::Launch(const base::FilePath& tor_bin,
     args.AppendArgPath(tor_watch_dir.AppendASCII("controlport"));
     args.AppendArg("--cookieauthentication");
     args.AppendArg("1");
+    args.AppendArg("--cookieauthfile");
```

Comment:
I'm confused. According to https://www.torproject.org/docs/tor-manual.html.en, cookieauthfile overrides the default location. so this basically changes `user_dir/tor/data/control_auth_cookie` to `user_dir/tor/watch/control_auth_cookie`, how does that help?

Reply:
We need to be notified when control_auth_cookie changes.  This change puts it in the directory we're watching for changes.

Reply:
(It is no longer necessary for controlport to be in the watch directory, but except for a handful of spurious wakeups it doesn't hurt either.)